src/app/cdk/panel/panel/panel.component.ts

```diff
@@ -87,6 +88,8 @@ export class PanelComponent implements OnInit {
   @Input() selectAll = false
   @Input() checkbox = false
   @Input() panelTitle: string
+  @Input() hasFeaturedIndex: boolean = false
```

Comment:
Do we really need both the `hasFeaturedIndex` input and the `featured-works` type input? Could we keep only one of those?